### PR TITLE
Update user-guide.md

### PR DESCRIPTION
**Pull request by humblec**
Correct snapshot deletion command.

### CODE REVIEW

File: snapshot/doc/user-guide.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ subjects:
 - kind: User
   name: alice
 ```
-This is only an example of API access configuration. Note the VolumeSnapshot objects behave just like any other Kubernetes API objects. Please refer to the [API access control documentation](https://kubernetes.io/docs/admin/accessing-the-api/) for complete guide on managing the API RBAC.
\ No newline at end of file
+This is only an example of API access configuration. Note the VolumeSnapshot objects behave just like any other Kubernetes API objects. Please refer to the [API access control documentation](https://kubernetes.io/docs/admin/accessing-the-api/) for complete guide on managing the API RBAC.
```
Comment (tsmetana) [inline snapshot/doc/user-guide.md]: OK. There was no newline at the end of the file...

File: snapshot/doc/user-guide.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ A Persistent Volume will be created and bound to the Persistent Volume Claim. Th
 ## Deleting Snapshot
 A Volume Snapshot `snapshot-demo` can be deleted as shown below:
 ```
-$ kubectl delete -f volumesnapshot/snapshot-demo
+$ kubectl delete volumesnapshot/snapshot-demo
```
Comment (tsmetana) [inline snapshot/doc/user-guide.md]: This is still not correct. Should be `kubectl delete volumesnapshot snapshot-demo`...

Comment (humblec) [reply snapshot/doc/user-guide.md]: @tsmetana Its not wrong, above syntax works :)

Comment (tsmetana) [reply snapshot/doc/user-guide.md]: Funny... I never really used it that way. OK then.

**Pull request merged**